packages/features/auth/lib/next-auth-options.ts

```diff
@@ -34,11 +39,19 @@ import CalComAdapter from "./next-auth-custom-adapter";
 import { verifyPassword } from "./verifyPassword";
 
 const log = logger.getSubLogger({ prefix: ["next-auth-options"] });
+
 const GOOGLE_API_CREDENTIALS = process.env.GOOGLE_API_CREDENTIALS || "{}";
 const { client_id: GOOGLE_CLIENT_ID, client_secret: GOOGLE_CLIENT_SECRET } =
   JSON.parse(GOOGLE_API_CREDENTIALS)?.web || {};
 const GOOGLE_LOGIN_ENABLED = process.env.GOOGLE_LOGIN_ENABLED === "true";
 const IS_GOOGLE_LOGIN_ENABLED = !!(GOOGLE_CLIENT_ID && GOOGLE_CLIENT_SECRET && GOOGLE_LOGIN_ENABLED);
+
+const IS_PASSKEY_LOGIN_ENABLED = !!(
+  process.env.PASSKEY_LOGIN_ENABLED === "true" &&
+  process.env.NEXT_PUBLIC_HANKO_PASSKEYS_TENANT_ID &&
+  process.env.HANKO_PASSKEYS_API_KEY
+);
+
```

Comment:
I think we already have this in constants - lets put it from there

Reply:
We do, but the `GOOGLE` variables are also duplicated in next-auth-options, so I did the same to stay consistent. Do you know if there's any reason for this?

Otherwise we could import both from constants.ts.
(I don't think the google constants differ from the ones in next-aut-options.ts in any way)